packages/dynamic-forms-bootstrap/styles/_defaults.scss

```
@@ -0,0 +1,71 @@
+// ============================================
+// BOOTSTRAP FORM DEFAULTS
+// ============================================
+// Opinionated host-framework defaults for Bootstrap apps.
+//
```

Stylelint (17.11.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)